db/migrations/20200430103648_add_non_canonical_column_to_storage_diff.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+ALTER TABLE public.storage_diff
+    ADD COLUMN non_canonical BOOLEAN NOT NULL DEFAULT FALSE;
+
```

Comment:
I think we should leave them as is for now and take care of them the next time we're processing that far back - since it's possible they could be untransformed for other reasons